src/tests/ftest/util/verify_perms.py

```diff
@@ -261,7 +261,10 @@ def _real_w(entry_type, path):
     if entry_type == 'file':
         try:
             # Write a newline to reduce likelihood of corrupting an executable file
-            with open(path, 'a', encoding='utf-8') as file:
+            with open(path, "w", encoding='utf-8') as file:
+                # O_APPEND is not supported by DFS, so "a" is not used in open. Need seek() here
+                # to move file pointer to the end of the file.
+                file.seek(0, 2)
```

Comment:
This is a bug-fix and un-related to the rest of the PR I assume?

Reply:
Right. It is from https://github.com/daos-stack/daos/pull/13200. Copied here to let CI finish all tests.